tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd

```diff
@@ -1,16 +1,16 @@
 ---
-syncID: 8346bef7292b46a09a76a0171b05662c
 title: "Getting Started with the R Programming Language"
-description: "This tutorial presents the basics of using R."
-dateCreated: 2014-11-04
-authors: Leah A. Wasser - Adapted from Software Carpentry
+code1: /R/R-basics/R-Basics-Getting-Started.R
 contributors: Donal O'Leary
```

Comment:
Sorry I missed this before. Please add yourself as a contributor to anything that you touched!